site/en/docs/extensions/reference/commands/index.md

```diff
@@ -23,11 +23,15 @@ or Ctrl, but is not required. Modifiers (such as Ctrl) can not be used in combin
 Keys. Tab key was removed from list of supported keys in Chrome version 33 and above for
 accessibility reasons.
 
-\* Please note that on Mac 'Ctrl' is automatically converted to 'Command'. If you want 'Ctrl'
-instead, please specify 'MacCtrl' under `"mac"`. Specifying 'MacCtrl' under `"default"` will cause
-the extension to be uninstallable.
+{% Aside %}
```

Comment:
This `Note: . . . {% Aside %} Please note that` stack is unfortunate. Maybe change this to two Aside notes instead, or a note and a warning.

Better suggestion: Remove "Note:" from the preceding paragraph, since it's really just some further information. And then maybe push the preceding TWO paragraphs into alignment like maybe:

```
**Supported keys** include A-Z, 0-9, Comma, Period, Home, End, PageUp, PageDown, Space, Insert, Delete, Arrow keys (Up, Down, Left, Right) and the Media Keys (MediaNextTrack, MediaPlayPause, MediaPrevTrack, MediaStop).

**Key combinations** must always include either Ctrl* or Alt. Combinations that involve Ctrl+Alt are not permitted in order to avoid conflicts with the AltGr key. Shift can be used in addition to Alt or Ctrl, but is not required. Modifiers (such as Ctrl) can not be used in combination with the Media Keys. Tab key was removed from list of supported keys in Chrome version 33 and above for accessibility reasons.
```

...or use definition list markdown for a more semantically explicit approach.

Reply:
TBH I didn't read this section closely when I first edited it. On a closer look, I'm not wild about it. I spent longer than I probably should have revising the entire intro.